release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -692,6 +692,22 @@ A platform Operator is an OLM-based Operator that can be installed during or aft
 
 For more information on platform Operators, see xref:../operators/admin/olm-managing-po.adoc#olm-managing-po[Managing platform Operators]. For more information on RukPak and its resources, see xref:../operators/understanding/olm-packaging-format.adoc#olm-rukpak-about_olm-packaging-format[Operator Framework packaging format].
 
+[id="ocp-4-12-olm-operator-node-affinity"]
+==== Controlling where an Operator is installed
+
+By default, when you install an Operator, OpenShift Container Platform installs the Operator pod to one of your worker nodes randomly.
+
+In {product-title} 4.12, you can control where an Operator pod is installed by adding affinity constraints to the Operator’s `Subscription` object.
+
+For more information, see xref:../operators/admin/olm-adding-operators-to-cluster.adoc#olm-overriding-operator-pod-affinity_olm-adding-operators-to-a-cluster[Controlling where an Operator is installed].
+
+[id="ocp-4-12-olm-pod-security-label-synchronization"]
+==== Pod security admission synchronization for user-created openshift-* namespaces
```

Comment:
Should this be in backticks since it's a literal [object reference](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-object-references)?
```suggestion
==== Pod security admission synchronization for user-created `openshift-*` namespaces
```

Reply:
We are not supposed to put markup (including literals) in headings. ([repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assemblymodule-titles-and-section-headings)) Though it often occurs in the docs.

Reply:
gotcha, is there a way to refer to it in some other way? E.g. `apiserver` can be referred to as "API server". The way it's presented currently looks sort of bizarre, but I'm not sure there's a good change to be made here.

Reply:
It is weirdly specific. The syncing does not occur for system created `openshift-` prefixed namespaces, only user-created ones. I debated going with _OpenShift prefixed namespaces_ but then it becomes inaccurate.